lib/jekyll/converters/markdown/kramdown_parser.rb

```diff
@@ -32,22 +32,44 @@ def setup
           @config["syntax_highlighter_opts"] ||= {}
           @config["coderay"] ||= {} # XXX: Legacy.
           modernize_coderay_config
+          make_accessible
         end
 
         def convert(content)
           Kramdown::Document.new(content, @config).to_html
         end
 
+        private
+        def make_accessible(hash = @config)
+          proc_ = proc { |hash_, key| hash_[key.to_s] if key.is_a?(Symbol) }
```

Comment:
We don't, it's because Kramdown wants symbols and we want strings.  So we give the best of both worlds so we don't disrupt our behavior and they can access it how they wish to without affecting us and us blindly accepting symbols.